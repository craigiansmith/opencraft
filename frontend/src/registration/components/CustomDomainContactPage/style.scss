@import '~styles/theme';

.app-container {
  min-height: 100vh;

  .app-main {
    flex-grow: 1;

    .custom-domain-contact-page {
      display: flex;
      flex-direction: column;
      align-items: stretch;
      flex: 1 1;

      h1 {
        text-align: center;
        color: #fff;
        font-family: $playfair-font;
        font-weight: bold;
        font-size: 40px;
        margin-bottom: 40px;
      }

      .custom-domain-contact-inner {
          background-color: #fff;
          flex-grow: 1;
          display: flex;
          flex-direction: column;
          align-items: center;
          padding-top: 60px;
          font-family: Chivo;
            font-size: 18px;
            letter-spacing: 0.54px;
            color: #0f1f24;

          .back-link {
            text-align: center;
            cursor: pointer;
            text-decoration: underline;
            font-family: "Chivo", sans-serif;
            font-size: 18px;
            letter-spacing: 0.54px;
            color: #1e4c59;
            margin-top: 40px;
          }
      }
    }
  }
}
