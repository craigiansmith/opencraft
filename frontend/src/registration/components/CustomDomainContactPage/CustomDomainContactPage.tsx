import { ROUTES } from 'global/constants';
import * as React from 'react';
import { NavLink } from 'react-router-dom';
import './style.scss';

interface Props {}

export const CustomDomainContactPage: React.FC<Props> = (props: Props) => {
  return (
    <div className="custom-domain-contact-page">
      <h1>Custom Domain</h1>
      <div className="custom-domain-contact-inner">
        <p>
          To use custom domain, please contact us at&nbsp;
          <a href="mailto:contact@opencraft.com">contact@opencraft.com</a>
        </p>
        <NavLink className="nav-link back-link" to={ROUTES.Registration.HOME}>
          Go back
        </NavLink>
      </div>
    </div>
  );
};
