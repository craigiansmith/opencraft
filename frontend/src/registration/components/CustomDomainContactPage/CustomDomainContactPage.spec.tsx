import React from 'react';
import { setupComponentForTesting } from 'utils/testing';
import { CustomDomainContactPage } from './CustomDomainContactPage';


describe("Custom Domain Contact Page", function() {
    it('Correctly renders Custom Domain Contact page', () => {
        const tree = setupComponentForTesting(
          <CustomDomainContactPage />
        ).toJSON();
        expect(tree).toMatchSnapshot();
    });
})

