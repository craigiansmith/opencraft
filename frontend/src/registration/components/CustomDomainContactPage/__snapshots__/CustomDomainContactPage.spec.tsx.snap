// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Custom Domain Contact Page Correctly renders Custom Domain Contact page 1`] = `
<div
  className="custom-domain-contact-page"
>
  <h1>
    Custom Domain
  </h1>
  <div
    className="custom-domain-contact-inner"
  >
    <p>
      To use custom domain, please contact -
      <a
        href="mailto:contact@opencraft.com"
      >
        contact@opencraft.com
      </a>
    </p>
    <a
      aria-current={null}
      className="nav-link back-link"
      href="/registration"
      onClick={[Function]}
    >
      Go back
    </a>
  </div>
</div>
`;
